mongo/document/validation_error.go

```diff
@@ -8,6 +8,9 @@ type ValidationErrors struct {
 }
 
 func (v *ValidationErrors) Error() string {
+	if v == nil {
+		return ""
```

Comment:
`return "empty validation error"` ?

LIke that it's easier to track if we get this error?